### DISCUSSION TITLE
JUNO模拟大作业中每个光子能量相同吗？

### DISCUSSION THREAD

**Discussion by mhliu0001** [physics-data/faq#77] [Q&A] [answered]
我看文档，里面没有提到每个顶点产生光子的能量问题。是否可以认为，液闪只会从一种激发态退激发到基态，产生一种能量的光子，所以每个光子的能量相同，都是1MeV/10000？

**Comment by Harry-Chen** [comment]
@Berrysoft

**Comment by Berrysoft** [comment, marked as answer]
液闪通常来说每个光子的能量~1MeV，这里为了好算取了10000的光产额，因此就这么算光子的能量也可以；实际上后面也强调说认为一个光子就是一个PE，所以光子的能量不是本题的重点。

**Comment by heroxbd** [comment]
Berrysoft ***@***.***> writes:

> 液闪通常来说每个光子的能量~1MeV，这里为了好算去了10000的光产额，因此就
> 这么算光子的能量也可以；

实际的过程是 1MeV 的能量的相当一部分成了热运动耗散了，一部分激发液闪分子
再退激成了这 10000 个可见光子。此处做了大量的近似，这里为了简化，我们就
不深究光子的能量和波长了，默认它们都是可见光。
